log-viewer/modules/TreeView.ts

```diff
@@ -204,20 +204,41 @@ function describeMethod(node: Method): Node[] {
 function renderBlock(line: LogLine) {
   const lineNode = divElem.cloneNode() as HTMLDivElement;
   lineNode.className = line instanceof Detail && line.hideable ? 'block name detail' : 'block name';
-
   lineNode.line = line;
 
-  const value = line.text || '';
-  let text = line.type + (value && value !== line.type ? ' - ' + value : '');
+  let valueElem;
+  if (line.value) {
```

Comment:
@kjonesffdc 
This block handles json formatted strings great.
Any ideas for apex objects? For example a USER_DEBUG like this
System.debug('here is an object: ' + myObj);
output:
'here is an object: (field1__c="field value", field2__c="field2", children__r=[{field1='field1'}])'

Trying to avoid writing a full blown parser to indent on new arrays object etc but I will if needed.